api/test/wfl/integration/modules/wgs_test.clj

```diff
@@ -16,8 +16,8 @@
 
 (clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
 
-(defn ^:private mock-really-submit-one-workflow [& _]
-  (UUID/randomUUID))
+(defn ^:private mock-submit-workflows [& _]
+  [(UUID/randomUUID)])
```

Comment:
Think this really needs to return as many UUIDs as sets of workflow inputs that are passed in.

Reply:
Good point!